apps/chat-ios/README.md

```diff
@@ -2,21 +2,21 @@
 
 ## Overview
 
-By completing the steps below, you will learn how to run the Amazon Chime SDK iOS chat demo applications on your mobile device or on a simulator. This application enables users to send and receive messages/image attachments in chat channels in real time. This application is designed in a way that can be used as a starting point for your own application, or as a quick way to explore the features of the Amazon Chime SDK messaging. Please see the [Build chat applications in iOS and Android with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging) for more information.
+This application is designed in a way that can be used as a starting point for your own application, or as a quick way to explore the features of the Amazon Chime SDK messaging. Please read blog posts below to start with various features.
+* [Build chat applications in iOS and Android with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-applications-in-ios-and-android-with-amazon-chime-sdk-messaging)
+* [Push Notifications for Android and iOS with Amazon Chime SDK messaging](https://aws-blogs-prod.amazon.com/business-productivity/push-notifications-for-android-and-ios-with-amazon-chime-sdk-messaging/)
 
 <p align="center">
 <img src="./media/awsChimeSDKMessagingDemo.png" alt="image" width="80%"/>
 </p>
 
 ## Prerequisites
 
-1. You have read [Build chat features into your application with Amazon Chime SDK messaging](https://aws.amazon.com/blogs/business-productivity/build-chat-features-into-your-application-with-amazon-chime-sdk-messaging/). You understand the architecture of the chat demo web application built with Amazon Chime SDK messaging. 
-2. You have completed the following operations
-    1. [Deploy the solution](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#deploying-the-solution).
-    2. [Run the Amazon Chime SDK Chat Demo](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#running-the-amazon-chime-sdk-chat-demo).
-    3. [Create Amazon Cognito Users](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#cognito-user-pools) OR [Use Credential Exchange Service](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#credential-exchange-service) to join a chat session.
-3. You have installed Xcode version 11.0 or later.
-4. You have [CocoaPods](https://cocoapods.org/) installed.
+1. [Deploy the AWS CloudFormation Template](https://github.com/aws-samples/amazon-chime-sdk/tree/main/apps/chat#deploying-the-solution).
```

Comment:
nit: title and tag seems mismatch `deploying-the-solution` vs `Deploy the AWS CloudFormation Template`